(#826) Update retention policies documentation and images. (#832)
chocolatey/docs: input/en-us/central-management/usage/website/administration/settings/retention.md

>
> This feature is available in Chocolatey Central Management starting with version 0.10.0.

In Chocolatey Central Management, a stale computer is one which hasn't reported in for a long period of time. This could be perfectly normal, but it could also be a sign that this computer is no longer active, and should be removed.  This is something that can be done manually, given the correct permissions, from the [computers page](xref:ccm-computers#removing-a-computer-from-central-management), however, enabling this setting automatically removes stale computers from Chocolatey Central Management.

By default, Stale Computer Deletion is disabled. When it is enabled, computers that haven't reported into Chocolatey Central Management within the set timeframe (the default is 365 days) will be removed.

![Stale Computer Deletion Settings](/assets/images/ccm/setup/website/stale-computer-deletion-settings.png)

## Deployment Plan Retention

> :choco-info: **NOTE**
>
> This feature is available in Chocolatey Central Management starting with version 0.11.0.

Chocolatey Central Management has the ability to automatically archive or delete deployments that have reached a completion state.

> By default, Deployment Plan Retention is disabled for both archiving and deleting. When it is enabled, deployment plans that have completed beyond the set timeframe (the default is 30 days) will be archived, or deleted. If both archive and delete is enabled, the delete period will be considered the combination of the archive and the delete.

Reviewer: ```suggestion
By default, Deployment Plan Retention is disabled for both archiving and deleting. When it is enabled, Deployment Plans that have completed beyond the set timeframe (the default is 30 days) will be archived, or deleted. If both archive and delete is enabled, the delete period will be considered the combination of the archive and the delete.

```

---
Order: 50
xref: ccm-administration-settings-retention

> Title: Retention Settings

Reviewer: Retention Policies

Author: ```suggestion
Title: Retention Policies
```

Description: Information about using retention settings within the Administration Settings screen
RedirectFrom:
  - en-us/central-management/usage/website/administration/settings/audit-retention
  - en-us/central-management/usage/website/administration/settings/stale-computer-deletion
---

Chocolatey Central Management has a number of settings that allow you to adjust various retention periods. By default, only the Audit Retention policy is enabled.

If you want to change these settings, follow these steps:

1. Open the CCM Site in the browser.
1. Login with the `ccmadmin` user.
1. In the left hand menu click on `Administration` and then `Settings`.

> 1. Click on the `Retention` tab

Reviewer: ```suggestion
1. Click on the `Retention Policies` tab

```

1. Modify the settings as required

> 1. Click the `Save All` button at the top right of the page to save your settings.

Reviewer: ```suggestion
1. Click the `Save All` button at the bottom left of the page to save your settings.

```
